source/_data/plugins/hexo-htmlnano-mini.yml

```diff
@@ -0,0 +1,9 @@
+description: A lightweight Hexo plugin to minify HTML using htmlnano
+link: https://github.com/RoversX/hexo-htmlnano-mini
+tags:
+  - html
+  - htmlnano
+  - minify
+  - css
+  - js
```

Comment:
Does `hexo-htmlnano-mini` will also work on css/js files?

Reply:
Yes. I initially tested this on [my blog](https://blog.closex.org) and resolved several issues during the process.

It now supports HTML minification using `htmlnano`, CSS minification with `clean-css`, and JavaScript minification with `uglify-js`. Additionally, it’s compatible with the `hexo-blog-encrypt` plugin. The default packages, `cssnano` and `Terser`, had some issues, so I replaced them with `clean-css` and `uglify-js` from `html-minifier`.
https://github.com/RoversX/hexo-htmlnano-mini/blob/main/package.json

```yml
htmlnano-mini:
  enable: true                        # 主开关
  enableInDev: false                  # 是否在开发服务器（hexo s）中启用压缩
  exclude: 
     #- '**/mycss.css'       
  collapseWhitespace: conservative    # 保守地压缩空白字符
  removeComments: safe                # 安全地移除注释
  removeEmptyAttributes: true         # 移除空属性 
  cleancss: true                      # 启用 clean-css 处理 CSS
  uglifyjs: true                      # 启用JavaScript 的压缩
```